src/lib/isDate.js

```diff
@@ -0,0 +1,35 @@
+function isValidFormat(format) {
+  return /(^(y{4}|y{2})[\/-](m{1,2})[\/-](d{1,2})$)|(^(m{1,2})[\/-](d{1,2})[\/-]((y{4}|y{2})$))|(^(d{1,2})[\/-](m{1,2})[\/-]((y{4}|y{2})$))/gi.test(format);
```

Comment:
I'd thought we'll come up with multiple regex depending with the format provided, starting with `YYYY/MM/DD` as a default.

We said we'll keep it simple as just date string only, nothing else.